python/jiminy_py/src/jiminy_py/viewer.py

```diff
@@ -551,6 +551,7 @@ def close(self=None):
             if self is None:
                 self = Viewer
             else:
+                Viewer._backend_robot_names.remove(self.robot_name)
```

Comment:
add comment "Consider that the robot is now available, no matter whether the robot has actually been deleted or not"